src/cli/main.ts

```diff
@@ -552,6 +552,8 @@ async function runCommand(command: string, toolArgs: any, optionArgs: any) {
       targetVersion,
       parsedOptionArgs
     );
+  } else if (command === "upgradeLegacySplats") {
```

Comment:
The `combine` command calls `combine`.
The `merge` command calls `merge`.
The `upgrade` command calls `upgrade`.
The `upgradeLegacySplats` command calls `upgradeSplats`.

Not "important", but ... maybe the command should be called `upgradeSplats`...?

More important: Documentation for this new command will have to be added into the main `README.md`, e.g. after the `upgrade` documentation, at https://github.com/CesiumGS/3d-tiles-tools/blob/24bf3b8a90074b5095f51bfaafbb5f2dbf11cf44/README.md?plain=1#L132 

(This may happen at any point in time, and it may make sense to do it "last", after the review. If you had already done it, there would be two places where the command may have to be renamed)